src/libs/antares/study/scenario-builder/TSnumberData.h

```diff
@@ -314,8 +310,49 @@ inline CString<512, false> ntcTSNumberData::get_prefix() const
     return "ntc,";
 }
 
-} // namespace ScenarioBuilder
-} // namespace Data
-} // namespace Antares
+// =================================
+// Transmission capacities ...
+// =================================
+
+class BindingConstraintsTSNumberData : public TSNumberData
+{
+public:
+    BindingConstraintsTSNumberData() = default;
+    virtual ~BindingConstraintsTSNumberData() = default;
+
+    bool reset(const Study& study) override;
+    void saveToINIFile(const Study& study, Yuni::IO::File::Stream& file) const override;
+
+    void setData(const std::string& group_name, unsigned year, unsigned value);
+    unsigned get(const std::string& group_name, unsigned year) const;
+    bool apply(Study& study) override;
+    CString<512, false> get_prefix() const override;
+    unsigned get_tsGenCount(const Study& study) const override;
+private:
+    std::map<std::string, MatrixType> rules_;
+};
+
+inline unsigned BindingConstraintsTSNumberData::get(const std::string& group_name, const unsigned year) const
+{
+    auto it = rules_.find(group_name);
+    if (it == rules_.end()) {
+        return 0;
+    }
+    return it->second[0][year];
+}
+
+inline CString<512, false> BindingConstraintsTSNumberData::get_prefix() const
+{
+    return "bc,";
+}
+
+inline unsigned BindingConstraintsTSNumberData::get_tsGenCount(const Study& study) const {
+    auto& parameters = study.parameters;
+
+    bool genTS = (0 != (parameters.timeSeriesToGenerate & timeSeriesBindingConstraints));
+    return genTS ? 1 : 0u;
+}
```

Comment:
`timeSeriesToGenerate & timeSeriesBindingConstraints` should always be 0, no TS is generated for BCs